api/src/main/java/org/pmiops/workbench/firecloud/RawlsConfig.java

```diff
@@ -0,0 +1,124 @@
+package org.pmiops.workbench.firecloud;
```

Comment:
the issues is we have too many packages/directories. The way we organize packages is messy and problematic. 
I was thinking using `firecloud` dir for most firecloud services, instead of kept of creating new ones. Just like we did for `google` package. There would be few more ApiClients coming to that dir soon, those would be small, and I don't think it works to have a different directories.